conduct.md

```diff
@@ -116,7 +116,11 @@ TODO: Write an introductory paragraph.
 
 ### Conflicts
 
-TODO: Write an introductory paragraph.
+When a conflict between students arise, it is imperative to deal with the situation in a respectful, yet effective way.
+Communication is a great tool to prevent or defuse conflicts between students, team members, or teams as a whole. According
+to Conway's Law the final product mirrors the methods of communication that went into the product's development. So if
+conflicts occur and prevent communication, it will likely lower the quality of the final product. For these reasons it is
```

Comment:
A comma should be added after "For these reasons"

Reply:
Thank you, your suggestions have been implemented!